### PR TITLE
Extract Bedrock-specific code to bedrock-veres-one-context.

### PR DESCRIPTION
**Pull request by dmitrizagidulin**
Addresses #2.

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -1,15 +1,11 @@
 /*
- * Copyright (c) 2017 Digital Bazaar, Inc. All rights reserved.
+ * Copyright (c) 2017-18 Digital Bazaar, Inc. All rights reserved.
```
Comment (mattcollier) [inline lib/index.js]: Use full year `2018`

**Review by mattcollier — APPROVED** [review]
One nit, and then needs review from @dlongley

File: lib/index.js (hunk 1 above)
Comment (dmitrizagidulin) [reply lib/index.js]: Done, thanks :)

File: README.md [hunk 2]
```diff
@@ -1 +1,42 @@
...
+```
+npm install veres-one-context
+```
+
+## Usage
+
+```
+const contextV1 = require('veres-one-context')
+
+contextV1['@context']['@version']  // -> "1.1"
```
Comment (dlongley) [inline README.md]: What's this about? This seems confusing ... this `@version` is referring to JSON-LD and not veres one, but this text makes it seem like it's telling the reader it's referring to veres one.

**Review by dlongley — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (dmitrizagidulin) [reply README.md]: Hehe, sorry, was just trying to demonstrate that it's parsed JSON, so I picked the very first properties in the object. I'll use something else. :)

Comment (dmitrizagidulin) [reply README.md]: There, changed, is that slightly less confusing?

**Review by dlongley — APPROVED** [review]

**Pull request merged**